test/integration/LoggingAdaptedConcepts.cpp

```diff
@@ -81,6 +99,9 @@ int main()
   BINLOG_INFO("{}", Wrap<int>{456});
   // Outputs: Wrap{ a: 456 }
 
+  BINLOG_INFO("{}", NsFoo{});
+  // Outputs: Stringable{ ns_str: bar }
```

Comment:
Should it be outputting `ns::Stringable...`?

Reply:
Fixed